content/tracing/advanced_usage/_index.md

```diff
@@ -1462,29 +1474,41 @@ import datadog.trace.api.CorrelationIdentifier;
 try {
     MDC.put("dd.trace_id", String.valueOf(CorrelationIdentifier.getTraceId()));
     MDC.put("dd.span_id", String.valueOf(CorrelationIdentifier.getSpanId()));
-} finally {
+} 
+
+// Log something
+
+finally {
     MDC.remove("dd.trace_id");
     MDC.remove("dd.span_id");
 }
 ```
 
+Then update your logger configuration to include `dd.trace_id` and `dd.span_id` in your log pattern:
+
+```
+<Pattern>"%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %X{dd.trace_id:-0} %X{dd.span_id:-0} - %m%n"</Pattern>
+```
+
+[See our Java logging documentation][2] for more details about specific logger implementation or to learn how to log in JSON.
+
 [1]: https://logback.qos.ch/manual/mdc.html
-[2]: https://docs.datadoghq.com/logs/log_collection/java/?tab=log4j#configure-your-logger
+[2]: https://docs.datadoghq.com/logs/log_collection/java/?tab=log4j#raw-format
 [3]: https://docs.datadoghq.com/tracing/languages/java/#configuration
 {{% /tab %}}
 {{% tab "Python" %}}
 
-You have two ways to inject trace information into your logs in Python:
+Follow one the two following option to inject trace information into your logs in Python:
 
-1. Automatically: Set the environment variable `DD_LOGS_INJECTION=true` when using `ddtrace-run`.
-2. Manually: Patch your `logging` module by updating your log formatter to include the ``dd.trace_id`` and ``dd.span_id`` attributes from the log record.
+**1. Automatic Trace Ids Injection**
 
-This integration with logs works if the log format includes:
+Set the environment variable `DD_LOGS_INJECTION=true` when using `ddtrace-run`.
 
-- `dd.trace_id=%(dd.trace_id)s`
-- `dd.span_id=%(dd.span_id)s`
+**2. Manual Trace Ids Injection**
 
-For instance: 
+Patch your `logging` module by updating your log formatter to include the ``dd.trace_id`` and ``dd.span_id`` attributes from the log record.
+
+The below configuration is the one used by the automatic injection method and is supported by default by the Log python integration:
```

Comment:
```suggestion
The configuration below is used by the automatic injection method and is supported by default in the Python Log Integration:
```

I'm taking a guess at "log python integration" here, tbh - but the original feels weird.